src/components/sections/items/ToggleSectionItem.tsx

```diff
@@ -61,13 +64,18 @@ export function ToggleSectionItem({
       style={topContainer}
       accessible={true}
       accessibilityRole="switch"
-      accessibilityState={{checked: checked}}
+      accessibilityState={{checked: checked, disabled: disabled}}
       accessibilityActions={[{name: 'activate'}]}
       onAccessibilityAction={() => onChange(!checked)}
+      accessibilityHint={
+        disabled ? t(SectionTexts.toggleInput.disabled) : undefined
+      }
```

Comment:
Should not be necessary to add this disabled hint. The accessibility state `disabled` should add the phrase "dimmed"/"nedtonet" to signal the status (if it doesn't, something is changed in recent RN versions).

We want to lean on the iOS/Android handling of disabled states rather than roll our own. Also we asked in a user test whether the phrase "nedtonet" was understood:
https://mittatb.slack.com/archives/CSCD9JWGP/p1622190495025200

Reply:
Aha, I didn't realize voiceover said "dimmed", so I thought the field wasn't read out at all 🙈